extensions/positron-catalog-explorer/package.json

```diff
@@ -0,0 +1,195 @@
+{
+  "name": "positron-catalog-explorer",
+  "displayName": "Catalog Explorer",
+  "description": "Explore tables, views, and files backed by data catalogs.",
+  "version": "0.0.1",
+  "publisher": "positron",
+  "icon": "resources/icon.png",
+  "private": true,
+  "engines": {
+    "vscode": "^1.87.0"
+  },
+  "categories": [
+    "Data Science"
+  ],
+  "activationEvents": [
+    "onStartupFinished"
+  ],
+  "main": "./out/extension.js",
+  "contributes": {
+    "configuration": {
+      "title": "Catalog Explorer",
+      "properties": {
+        "catalogExplorer.enabled": {
+          "type": "boolean",
+          "default": false,
+          "markdownDescription": "**Experimental:** Enable or disable the experimental Catalog Explorer extension. When disabled, the extension will not activate.",
```

Comment:
Just to confirm, since this has the "experimental" tag, we can remove `**Experimental:**` from the description here.

Reply:
updated!